laravel/tests/Unit/EvaluationParserTest.php

```diff
@@ -119,19 +150,25 @@ public function testGetMinutesDiff() {
 // Tests for getTime()
 
     public function testGetTimeForLineWithDate() {
-        $time = EvaluationParser::getTime("03/22/21 1930", "2021-03-22");
+        $parser = new EvaluationParser('20210323', true);
```

Comment:
I may be missing something obvious, but doesn't `EvaluationParser` only take one argument?